### ISSUE TITLE
apt-dater is still saying 'R' after rebooting machines

### ISSUE THREAD

**Issue by MAGICCC** [DE-IBH/apt-dater#126]
Hi,

I installed apt-dater on my VMs today, but after install kernel updates and rebooting the VMs apt-dater is still saying that I should reboot my VMs...
![728](https://cloud.githubusercontent.com/assets/3279213/24840571/a81b143a-1d6f-11e7-9f52-a85110751f63.png)

**Comment by liske** [comment]
The **R**eboot flag depends on the result of *apt-dater-host* on the target server which uses *needrestart* to check for outdated kernels. Since Ubuntu picks *apt-dater* and *needrestart* packages w/o any QA they are often somehow broken.

**Comment by MAGICCC** [comment]
So tl;dr this output can't be used for 'hey please reboot me' anymore

**Comment by liske** [comment]
It can, but you need to have a reasonable apt-dater-host+needrestart versions on the targets where Ubuntu is known to have QA problems.

**Comment by nosilver4u** [comment]
So, is there anything that can be done about this? I tried grabbing newer versions of apt-dater-host and needrestart from launchpad (the ones for zesty/16.10), which appear to match the versions I'm running on Debian 9, but the R flag does not go away.

These are what I downloaded and installed:
https://launchpad.net/ubuntu/+archive/primary/+files/needrestart_2.11-2_all.deb
https://launchpad.net/ubuntu/+archive/primary/+files/apt-dater-host_1.0.0-4_all.deb

The output from running the 'needrestart' command appears to match what my Debian systems display, but apt-dater still complains that it needs a reboot.
